code/modules/mob/living/simple_animal/hologram/hologram.dm

```diff
@@ -97,6 +97,10 @@
 	head = null
 	w_uniform = null
 	wear_suit = null
+	for (var/i = 1 to held_items.len)
+		var/obj/item/O = held_items[i]
```

Comment:
Why not just `for(var/obj/item/O in held_items)`? That would also remove the need for the `if(O)` below because of the implicit typechecking